plugin.video.jafc/main.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+
+from resources.lib import plugin
+
```

Comment:
Please insert `if __name__ == '__main__':` to avoid unwanted code execution.

Reply:
This seems an odd request - I have submitted and had accepted 3 plugins that don't have this, moreover on looking at other plugins in the repo almost none have this either. I can also see no mention of it in the plugin dev notes...